app/tests/src/test_ingest_guru_cards.py

```diff
@@ -39,15 +39,22 @@ def guru_s3_file(mock_s3_bucket_resource, sample_cards):
     return "s3://test_bucket/guru_cards.json"
 
 
+doc_attribs = {
+    "dataset": "test_dataset",
+    "program": "test_benefit_program",
+    "region": "Michigan",
+}
+
+
 @pytest.mark.parametrize("file_location", ["local", "s3"])
-def test__ingest_cards_local(db_session, guru_local_file, guru_s3_file, file_location):
+def test__ingest_cards(db_session, guru_local_file, guru_s3_file, file_location):
     db_session.execute(delete(Document))
     mock_embedding = MockSentenceTransformer()
 
     if file_location == "local":
-        _ingest_cards(db_session, mock_embedding, guru_local_file)
+        _ingest_cards(db_session, mock_embedding, guru_local_file, doc_attribs)
     else:
-        _ingest_cards(db_session, mock_embedding, guru_s3_file)
+        _ingest_cards(db_session, mock_embedding, guru_s3_file, doc_attribs)
 
     documents = db_session.execute(select(Document).order_by(Document.name)).scalars().all()
     assert len(documents) == 3
```

Comment:
Suggest add these assertions to verify doc_attribs is being processed as we expect:
```suggestion
    assert len(documents) == 3
    assert set(d.dataset for d in documents) == {"test_dataset"}
    assert set(d.program for d in documents) == {"test_benefit_program"}
    assert set(d.region for d in documents) == {"Michigan"}
```